mappings/net/minecraft/world/border/WorldBorder.mapping

```diff
@@ -94,3 +97,39 @@ CLASS net/minecraft/class_2784 net/minecraft/world/border/WorldBorder
 		METHOD <init> (Lnet/minecraft/class_2784;D)V
 			ARG 1 size
 		METHOD method_11996 recalculateBounds ()V
+	CLASS class_5200 Configuration
```

Comment:
We usually use `Config` in Yarn (see `FeatureConfig`, `DecoratorConfig` etc)

Reply:
Ah, right.

Reply:
All sorted. Thank ye!